org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/CompletionHandlerTest.java

```diff
@@ -1290,7 +1290,7 @@ public void testCompletion_getter() throws Exception {
 		assertNotNull(resolvedItem.getTextEdit());
 		assertTextEdit(2, 4, 7, "/**\n" +
 				 " * @return the strField\n" +
-				 " */\n" +
+				 " */\n\n" +
```

Comment:
this extra line is definitely wrong. Eclipse doesn't have it. Where is it coming from?

Reply:
In GetterSetterUtil, getGetterStub(), and getSetterStub(), The JDT version has it and JDT-LS removed it.

```
if (comment != null) {
  buf.append(comment);
- buf.append(lineDelim);
}
```
However, upon actually testing (create a field, hover over it, create getter/setters), the results is the same as before. This is why I was guessing that everything is ultimately fed to a formatter which corrects the issue.